### DISCUSSION TITLE
Who are able to look at ImageBrush issues?

### DISCUSSION THREAD

**Discussion by FrozDark** [unoplatform/uno#9712] [Q&A]
Sorry for asking that for you but I would like to resolve the issues related to ImageBrush as I'm fully migrating to Uno
https://github.com/unoplatform/uno/issues/9537
https://github.com/unoplatform/uno/issues/9549

Is anybody have time to look into them? Thanks

**Comment by jeromelaban** [comment]
The Uno Platform team generally work based on paying clients priorities. Let us know what you need by contacting us at info@platform.uno.
